api/src/main/java/org/pmiops/workbench/exfiltration/jirahandler/EgressVwbJiraHandler.java

```diff
@@ -0,0 +1,126 @@
+package org.pmiops.workbench.exfiltration.jirahandler;
+
+import jakarta.inject.Provider;
+import java.time.Instant;
+import java.util.Optional;
+import java.util.logging.Logger;
+import java.util.stream.Stream;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.db.model.DbEgressEvent;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.db.model.DbWorkspace;
+import org.pmiops.workbench.exfiltration.EgressRemediationAction;
+import org.pmiops.workbench.exfiltration.ExfiltrationUtils;
+import org.pmiops.workbench.jira.ApiException;
+import org.pmiops.workbench.jira.JiraContent;
+import org.pmiops.workbench.jira.JiraService;
+import org.pmiops.workbench.jira.model.AtlassianContent;
+import org.pmiops.workbench.jira.model.SearchResults;
+import org.pmiops.workbench.model.EgressEvent;
+import org.pmiops.workbench.utils.mappers.EgressEventMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service(ExfiltrationUtils.EGRESS_VWB_JIRA_HANDLER_QUALIFIER)
+public class EgressVwbJiraHandler extends EgressJiraHandler {
+
+  private static final Logger log = Logger.getLogger(EgressVwbJiraHandler.class.getName());
+
+  private final Provider<WorkbenchConfig> workbenchConfigProvider;
+  private final EgressEventMapper egressEventMapper;
+
+  @Autowired
+  public EgressVwbJiraHandler(
+      Provider<WorkbenchConfig> workbenchConfigProvider, EgressEventMapper egressEventMapper) {
+    this.workbenchConfigProvider = workbenchConfigProvider;
+    this.egressEventMapper = egressEventMapper;
+  }
+
+  /**
+   * File a Jira investigation ticket if no open ticket exists for this VM. Identify matching
+   * tickets by the "Egress VM Prefix" and "RW Environment" custom fields on the ticket. If a ticket
+   * already exists, add a comment instead.
+   */
+  public void logEventToJira(DbEgressEvent event, EgressRemediationAction action)
+      throws ApiException {
+    String envShortName = workbenchConfigProvider.get().server.shortName;
+    SearchResults results = searchJiraIssuesWithLabel(event, envShortName, "high-egress");
+    log.info(
+        String.format("Found %d jira issues with label: high-egress", results.getIssues().size()));
+    if (results.getIssues().isEmpty()) {
+      createJiraIssueWithLabels(event, action, envShortName, new String[] {"high-egress"});
```

Comment:
in addition to high-egress label, should we add another label for vwb tickets?

Reply:
ah, okay, I also add vwb label then. Does that sound good?

Reply:
yea I think it will make it easier to filter on those